.github/workflows/publish.yml

```diff
@@ -0,0 +1,43 @@
+name: Publish
+on:
+  push:
+      branches: [master]
```

Comment:
After merge, we need to run the tests (`build-and-publish.yml`) before triggering this workflow. How are you planning to specify this dependency?

Reply:
Good call, I added a dependency on the build-and-test workflow